polymesh_schema.json

```diff
@@ -204,6 +204,19 @@
     "PermissionedValidator": {
         "compliance": "Compliance"
     },
+    "Authorization": {
+        "authorization_data": "AuthorizationData",
+        "authorized_by": "IdentityId",
+        "expiry": "Option<Moment>",
+        "next_authorization": "u64",
+        "previous_authorization": "u64"
+    },
+    "AuthorizationData": {
+        "_enum": {
```

Comment:
I'm not sure if that makes difference but `_enum` down at line 221 uses a list contained in `[ ]` rather than in `{ }`. Which of those two is the correct notation? Or is it both?

Reply:
Unfortunately, none of them is "correct". These are hacks to make polkadot.js work. With [] approach, enums that have arguments, don't work. with {} approach, enums that don't have arguments don't work.

Eventually, we'll not even need this file. It's a planned feature in polkadot.js to be able to fetch types directly from the node.